setup.cfg

```diff
@@ -0,0 +1,85 @@
+[mypy]
+python_version = 3.7
+
+[mypy-Bio.*]
+ignore_missing_imports = True
+
+[mypy-numpy]
+ignore_missing_imports = True
+
+[pylint]
+disable =
```

Comment:
There is a lot of stuff under `disable`. Could you add a comment or two explaining what things (broadly speaking) are being disabled or enabled?